tests/lean/tc_out_param_deps.lean

```diff
@@ -0,0 +1,23 @@
+prelude
+import init.core
```

Comment:
If you don't care about the output of the test, please move it in `tests/lean/run`.